implementors.md

```diff
@@ -3,11 +3,17 @@ layout: page
 title: Implementing the OpenTracing APIs
 ---
 
-OpenTracing makes tracing portable for those adding instrumentation to their production systems.
+### Why OpenTracing?
 
-That said, it also makes life easier for those implementing monitoring and tracing tools: rather than spending time and effort learning the idioms of numerous platforms, designing featureful tracing APIs, and maintaining them, projects which implement OpenTracing can focus on delivering value and making the most of the instrumented data.
+OpenTracing provides portable APIs for those adding instrumentation to their production systems.
 
-Furthermore, in a codebase instrumented with OpenTracing, adding (or switching to) an OpenTracing-compatible tool is an `O(1)` operation for the programmer: it's really just a configuration change. Since instrumentation is often the greatest obstacle to adoption, OpenTracing support is a great way to put a monitoring and/or tracing tool in the hands of more developers quickly.
+To make life easier for those implementing monitoring and tracing tools, OpenTracing handles the time-consuming process of designing and maintaining featureful tracings APIs across programming languages and respecting their various idioms. This allows tracing tool implementors to focus on delivering value and making the most of the instrumented data itself.
 
-For information about current reference implementations in particular platforms, please reach out to the OpenTracing authors via [email](https://groups.google.com/forum/#!forum/distributed-tracing) or
+In a codebase instrumented with OpenTracing, adding or switching to an OpenTracing-compatible tool becomes an `O(1)` operation for the programmer: it's just a configuration change - there's no need to modify the code instrumentation itself. Since the instrumentation process is often the greatest obstacle to adoption, building an OpenTracing-compatible implementation is a great way to get a monitoring and tracing tool into the hands of more developers more quickly.
+
+### How to Add an OpenTracing Implementation
+
+For more information about the current reference implementations for the various platforms, see the [OpenTracing GitHub page](https://github.com/opentracing).
```

Comment:
if we do get reference implementations, are we saying they are necessarily going to be in the opentracing org?  In theory we could designate 3rd party projects as reference.